bindings/java/src/main/com/apple/foundationdb/ClusterOptions.java

```diff
@@ -0,0 +1,5 @@
+package com.apple.foundationdb;

/**
+ * A set of options that can be set on a {@link Cluster}.
+ * 
+ * There are currently no options available.
+ */
public class ClusterOptions extends OptionsSet {
	public ClusterOptions( OptionConsumer consumer ) { super(consumer); }
}

```

Comment:
I think this was checked in because we no longer generate it, but it's used to support the old API. Agree that we should clean it up and deprecate it.